pkgs/racket-doc/scribblings/reference/contracts.scrbl

```diff
@@ -2044,7 +2059,9 @@ the export.
           (eval:error (recip 1+2i))]
 
 @history[#:changed "7.3.0.3" @list{Added @racket[#:unprotected-submodule].}
-         #:changed "7.7.0.9" @list{Started ignoring @racket[ignored-id].}]
+         #:changed "7.7.0.9" @list{Started ignoring @racket[ignored-id].}
+         #:changed "8.12.0.13" @list{Added @racket[contract-in]}
```

Comment:
I think the change to add the `8.12.0.13` entry should be added to `master` right now as a separate commit.